README.md

```diff
@@ -102,6 +102,13 @@ cargo run dump-website
 
 The website will automatically load new teams added here, however they cannot be translated unless `tools.ftl` is also updated.
 
+You can also print a list of users with individual access to repositories
+
+```
+cargo run dump-individual-access
```

Comment:
I think this is fine for now, but I have some other ideas for views on repos that we might want to do that makes me think we might want to think about a repo subcommand:

```
cargo run repo dump-individual-access
```